src/Controller/Back/SignalementActionController.php

```diff
@@ -117,13 +118,22 @@ public function addSuiviSignalement(
 
                 return $this->redirectToRoute('back_signalement_view', ['uuid' => $signalement->getUuid()]);
             }
-            $suivi->setDescription($content);
-            $suivi->setIsPublic(!empty($form['notifyUsager']));
-            $suivi->setSignalement($signalement);
-            $suivi->setCreatedBy($this->getUser());
-            $suivi->setType(Suivi::TYPE_PARTNER);
-            $doctrine->getManager()->persist($suivi);
-            $doctrine->getManager()->flush();
+            $suivi = (new Suivi())
+                ->setDescription($content)
+                ->setIsPublic(!empty($form['notifyUsager']))
+                ->setSignalement($signalement)
+                ->setCreatedBy($this->getUser())
+                ->setType(Suivi::TYPE_PARTNER);
+
+            try {
+                $suiviRepository->save($suivi, true);
```

Comment:
Je suis pas en phase avec la duplication de la méthode save dans le repository, voila c'est dit :)

Reply:
ah oui tiens, pourquoi donc cette méthode save dans un repo ? :)

Reply:
@numew il me semble que tu l'as déjà dit :-)

Reply:
ils on vite fait machine arrière https://github.com/symfony/maker-bundle/issues/1327

Reply:
Je comprend les arguments car ça reste un projet open source et tout ce que ça implique (nous on est 4 :-p  c'est censé être plus facile :-p)

Surtout cet argument
> it's confusing: one may thing it will flush only the new message (it's not!)

Il y'a du avoir pas mal de retour suite à une mauvaise compréhension du flush qui a aussi évolué car si je me souviens bien il fut un temps ou flush($entity) existait. 

Blague à part, je lis que qu'avant de s'appeler `save()`, ça s'appelait `add()` (Y'a pas toujours des bonnes idées dans la communauté :-p )

Pour finir du moment qu'il y'a une bonne compréhension du flush dans l'équipe, ça ne me pose pas de problème de le laisser disponible dans le repo mais ça reste une décision collégiale donc 2 contre 1 (je supprime :-) )

En attendant le ticket sur un nouveau create dans le manager->create()